team-config.yml

```diff
@@ -168,3 +168,21 @@ opal:
     build_notices_channel: "#opal-builds"
   tags:
     application: opal
+pdda:
+  team: "PDDA"
+  namespace: "pdda"
+  azure_ad_group: DTS SDS PDDA
+  slack:
+    contact_channel: "#pdda"
+    build_notices_channel: "#pdda-builds"
+  tags:
+    application: pdda
+pdm:
+  team: "PDM"
+  namespace: "pdm"
+  azure_ad_group: DTS SDS PDM
+  slack:
+    contact_channel: "#pdm"
+    build_notices_channel: "#pdm-builds"
+  tags:
+    application: pdm
```

Comment:
was this assigned to you? if it wasn't can you expand the acronym at least so people not familiar with your project know what it is

Reply:
Hi. PDM is the Public Display Manager currently part of XHIBIT, but is being migrated as a standalone application alongside PDDA (Public Display Data Aggregator). PDM is used to manage the public displays in the crown courts.
PDDA will handle the retrieval of data from Common Platform to be display on the public displays; currently this is also handled by XHIBIT.

Reply:
```suggestion
    application: public-display-manager
```